src/routes/contact.js

```diff
@@ -261,7 +262,7 @@ router.post('/send/:type/:id', async (req, res) => {
         entityId: entity.id,
         token: temporaryToken
       }
-      email.sendContactCheckInEmail(e).then(() => {
+      email.sendContactCheckInEmail(e, req.headers.origin).then(() => {
```

Comment:
We're sending this info in headers and not the body? This might require an adjustment on the API request. Maybe set an option to use an env var instead? So:
```
const fromAddress = (process.env.SEND_EMAIL !== undefined) ? process.env.SEND_EMAIL : req.headers.origin
email.sendContactCheckInEmail(e, fromAddress)
```


Probably need to adjust unit tests to ensure coverage here.